README.md

```diff
@@ -1,3 +1,21 @@
-# Ansible Collection - crowdstrike.crowdstrike
+# Ansible Collection - crowdstrike.falcon
 
-Documentation for the collection.
+This collection is focused on downloading and installing the CrowdStrike Falcon Sensor.
+It can also uninstall the Falcon Sensor. Other enhancements are possible in the future
+to add more functionality.
+
+# Installation
+
+To install the collection from Ansible Galaxy:
+
+```
+ansible-galaxy collection install crowdstrike.falcon
+```
+
+# Contributing
```

Comment:
Can we also give an example of how to use the role after downloading?

Reply:
@shawndwells done